public/app/stores/room/room-store.js

```diff
@@ -5,6 +5,46 @@ var roomActions = require('actions/room');
 var roomStore = Reflux.createStore({
   listenables: [roomActions],
 
+  getInitialState: function getInitialState () {
+    return [
+      {
+        title: 'Living room',
+        devices: [
+          {
+            id: '0',
+            name: 'yetu Home Gateway',
+            type: 'Home Gateway',
+            state: 'connected',
+            description: 'test description'
+          },
+          {
+            id: '1',
+            name: 'Nest',
+            type: 'Thermostat',
+            state: 'not conn.',
+            description: 'test description'
+          }
+        ]
+      },
+      {
+        title: 'Bed room',
+        devices: [
+          {
+            id: '2',
+            title: 'Nest',
+            type: 'Thermostat',
+            state: 'connected',
+            description: 'test description'
+          }
+        ]
+      },
+      {
+        title: 'Bath room',
+        devices: []
+      }
+    ];
+  },
+
```

Comment:
Because we don't have an API design yet, therefore a proper data model can't be created and fixtures are used. Since the stores are not connected in any way the data they manage has a degree of redundancy. We'll have a proper data flow when we have a clear API design from the infrastructure team.